cypress/e2e/listingDetails/machineTranslations.e2e.ts

```diff
@@ -11,7 +11,7 @@ const listings = {
     title: "TEST Automated Listing (do not modify)",
   },
   HABITAT_SALE: {
-    id: "a0W4U00000KnMyXUAV",
+    id: "a0W0P00000GlKfB",
     address: "36 Amber Drive, San Francisco, CA 94131",
     title: "Habitat Amber Drive",
   },
```

Comment:
since this is no longer habitat listing, should we rename the var, address, title?